recipes-sota/aktualizr/files/libcrypto_valgrind_supp.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/tests/aktualizr.supp b/tests/aktualizr.supp
+index 51aae5c5..2b65f5c0 100644
+--- a/tests/aktualizr.supp
++++ b/tests/aktualizr.supp
+@@ -90,3 +90,19 @@
+    fun:ostree_sysroot_simple_write_deployment
+    ...
+ }
++{
++   libcrypto.so.1.0.2 false positives (see e.g. https://github.com/scikit-hep/pyjet/blob/master/etc/valgrind-python.supp)
```

Comment:
Maybe link to something more explanatory, like https://github.com/curl/curl/issues/1117 or http://openssl.6102.n7.nabble.com/PURIFY-flag-in-md-rand-c-and-a-strongly-worded-comment-td41674.html.

Also, I don't think there will be an Openssl 1.0.3,  `obj:/usr/lib/libcrypto.so.1.0.*` shouldn't hurt